src/unit_tests/os_crypto/md5_sha1_sha256/CMakeLists.txt

```diff
@@ -8,7 +8,10 @@
 cmake_minimum_required(VERSION 3.10)
 
 list(APPEND md5_sha1_tests_names "test_md5_sha1_sha256_op")
-list(APPEND md5_sha1_tests_flags "-Wl,--wrap,wpopenv,--wrap,wpclose,--wrap,fread,--wrap,fclose,--wrap,fflush,--wrap,fgets,--wrap,fgetpos,--wrap,fopen,--wrap,fseek,--wrap,fwrite,--wrap,remove,--wrap,fgetc,--wrap,wfopen")
+list(APPEND md5_sha1_tests_flags "-Wl,--wrap,wpopenv -Wl,--wrap,wpclose -Wl,--wrap,fread -Wl,--wrap,fclose \
+                                  -Wl,--wrap,fflush -Wl,--wrap,fgets -Wl,--wrap,fgetpos -Wl,--wrap,fopen \
+                                  -Wl,--wrap,fseek -Wl,--wrap,fwrite -Wl,--wrap,remove -Wl,--wrap,fgetc \
+                                  -Wl,--wrap,wfopen -Wl,--wrap,popen -Wl,--wrap,_mwarn")
```

Comment:
The wrap for function "popen" is not needed in this test, since it is not used by the called functions in it.

Reply:
Yes, it is necessary because the `__wrap_popen` mock was modified to use test_mode, and it must be added so that it finds `__real_popen`, although it is not used directly, if there are `popen` calls in the test

Reply:
Ditto

Reply:
As discussed in the huddle, it was determined that they were necessary.